test/integration/crcsuite/crcsuite.go

```diff
@@ -175,6 +179,28 @@ func ConfigFileInCRCHomeContainsKey(format string, configFile string, condition
 	return nil
 }
 
+func LoginToOcClusterSucceedsOrFails(expected string) error {
+
+	bundle := strings.Split(bundleName, ".crcbundle")[0]
+	pswdLocation := filepath.Join(CRCHome, "cache", bundle, "kubeadmin-password")
+
+	pswdFile, err := os.Open(pswdLocation)
+	if err != nil {
+		return err
+	}
+	defer pswdFile.Close()
+
+	pswd, err := ioutil.ReadAll(pswdFile)
+	if err != nil {
+		return err
+	}
```

Comment:
You can directly use `ioutil.ReadFile` instead first opening the file and reading it.